ceph-dev-new-build/build/setup_rpm

```diff
@@ -100,6 +79,36 @@ get_rpm_dist() {
 
 get_rpm_dist
 
+if [ "$RELEASE" = 7 ]; then
+   if [ "$ARCH" = x86_64 ]; then
+       $SUDO yum install -y centos-release-scl
+   elif [ "$ARCH" = arm64 ]; then
+       $SUDO yum install -y centos-release-scl-rh
+       $SUDO yum-config-manager --disable centos-sclo-rh
+       $SUDO yum-config-manager --enable centos-sclo-rh-testing
+   fi
+elif [ "$RELEASE" = 8 ]; then
+    $SUDO dnf config-manager --set-enabled PowerTools
+    # chacractl is not python3 compatible yet
+    $SUDO dnf -y install python2
```

Comment:
see https://github.com/ceph/chacractl/pull/44 fix a fix. hopefully, we can get this fix included in next release soon.